docket.mk

```diff
@@ -1,12 +1,27 @@
-%_import : %.csv
+%_import : %.csv sfm_pc/management/commands/country_data sfm_download.zip
```

Comment:
Do you need `sfm_download.zip` for the normal import to run? If not, you can omit it here and add as dependency of the target to upload the archive to S3, later.

Reply:
Correct, it's like that in my other branch where I've implemented the upload.